pkg/kore/assets/plans.go

```
@@ -58,12 +58,14 @@ func GetDefaultPlans() []*configv1.Plan {
 								}
                             ],
 							"diskSize":               100,
+							"enableAutoupgrade":      true,
 							"enableAutorepair":       true,
 							"enableAutoscaler":       true,
 							"enableHTTPLoadBalancer": true,
 							"enableHorizontalPodAutoscaler": true,
 							"enableIstio": false,
 							"enablePrivateNetwork":   false,
+							"enableShieldedNodes": false,
```

This should be `true` so dev teams can fail early. Best practice.

---

Agreed and added ...